cmd/src/campaigns_create.go

```diff
@@ -126,3 +153,111 @@ const createcampaignMutation = `mutation CreateCampaign($input: CreateCampaignIn
   }
 }
 `
+
+const (
+	sep    = "------- EVERYTHING BELOW THIS LINE WILL BE IGNORED -------"
+	notice = `You are creating a new campaign.
+Write a name and description for this campaign in this file.
+The first line of text is the name and the rest is the description.`
+)
+
+type CampaignEditor struct {
+	Name        string
+	Description string
+}
+
+func (e *CampaignEditor) EditAndExtract() (string, string, error) {
+	f, err := ioutil.TempFile("", "new-campaign")
```

Comment:
can't you just give this file the suffix `.md` so all editors highlight it correctly?

Reply:
Saw this too late: https://github.com/sourcegraph/src-cli/pull/92